packages/radix/src/components/Code.tsx

```diff
@@ -12,14 +12,15 @@ export const Code = React.forwardRef<HTMLElement, CodeProps>((props, forwardedRe
       base: {
         code: {
           normal: {
-            fontFamily: theme.fonts.mono,
-            fontSize: theme.fontSizes[2],
-            lineHeight: theme.lineHeights[1],
-            paddingLeft: theme.space[1],
-            paddingRight: theme.space[1],
-            borderRadius: theme.radii[1],
-            color: theme.colors.gray800,
+            borderRadius: '0.25em',
             boxShadow: `inset 0 0 0 1px ${theme.colors.gray300}`,
+            color: theme.colors.gray800,
+            fontFamily: theme.fonts.mono,
+            fontSize: '80%',
+            lineHeight: 1,
```

Comment:
@benoitgrelard Can we put `line-height: 1` on the Code primitive? It currently inherits the line height, which may cause alignment issues. Monospace fonts usually have chunkier metrics, which may cause lines to shift when they are around other fonts because the browser will try to align their baselines (unless a different vertical-align is set, which would be even more horrendous).

<img src="https://user-images.githubusercontent.com/8441036/82591185-9fbc8f80-9ba7-11ea-87e9-d2e6a17b91d8.gif" width="613" />

Reply:
Currently your line height 1 will map to the scale. You know that right?

Reply:
Yeah I think adding line height 1 to Primitives is fine. Wanna leave it out of this PR If we agree to add it to Primitives?

Reply:
Yeah I'll be happy to leave it out

Reply:
let's get it

Reply:
Done in c50ed2249761e1ea50cbea2abd71c15b063955d9, tracking https://github.com/modulz/modulz/issues/1264